CMakeLists.txt

```diff
@@ -28,6 +28,12 @@ set(CMAKE_VERBOSE_MAKEFILE OFF)
 
 get_filename_component(REPOSITORY_DIR ${PROJECT_SOURCE_DIR} ABSOLUTE)
 
+# gcc v4.9 requires its own binutils-wrappers for LTO (flag -flto)
+# fixes #981
+IF(UNIX AND CMAKE_COMPILER_IS_GNUCXX AND (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "4.9" OR CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL "4.9"))
+    set(CMAKE_AR "/usr/bin/gcc-ar")
+    set(CMAKE_RANLIB "/usr/bin/gcc-ranlib")
```

Comment:
The absolute paths for "/usr/bin/gcc-ar" and "/usr/bin/gcc-ranlib" may be a bad idea, since it only works if the toolchain is installed using those absolute paths versus relying on the environment. Also, your changes in Apr1Lib.cmake and AprUtil1Lib.cmake don't use absolute paths.